From 7dc8a06f8a7ca34d7a11248c76621217d86f63ce Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Thu, 10 Dec 2020 01:20:11 +0100
Subject: [PATCH] turris-netboot: fix location of gen_initramfs_list.sh

In Linux kernel it was moved from scripts folder to usr.

Fixes:
bash: ./scripts/gen_initramfs_list.sh: No such file or directory
---
 utils/turris-netboot/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/turris-netboot/Makefile b/utils/turris-netboot/Makefile
index 34e87caed..963ac61de 100644
--- a/utils/turris-netboot/Makefile
+++ b/utils/turris-netboot/Makefile
@@ -84,7 +84,7 @@ endef
 
 define Package/turris-netboot-data/install
 	$(INSTALL_DIR) $(1)/usr/share/turris-netboot
-	cd $(LINUX_DIR); ./scripts/gen_initramfs_list.sh \
+	cd $(LINUX_DIR); ./usr/gen_initramfs_list.sh \
 	-o $(1)/usr/share/turris-netboot/initrd-$(CONFIG_ARCH).gz -u squash -g squash \
 	"$(PKG_BUILD_DIR)/rootfs" "$(PKG_BUILD_DIR)/rootfs.extra"
 	gzip -d $(1)/usr/share/turris-netboot/initrd-$(CONFIG_ARCH).gz
-- 
2.25.1

